### PR TITLE
OCPBUGS-38800: Added link to About the Kubernetes NMState Operator fo…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16+

Issue:
[OCPBUGS-38800](https://issues.redhat.com/browse/OCPBUGS-38800)

Link to docs preview:
[About the Kubernetes NMState Operator](https://80773--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html)

- [x] SME has approved this change (Will Russell)
- [x] QE has approved this change (Qiong Wang).

### CODE REVIEW

**Review by Scotchman0 — APPROVED** [review]
LGTM!

**Review by JoeAldinger — APPROVED** [review]
/lgtm

File: networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ Before you can use NMState with {product-title}, you must install the Kubernetes
...
+On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the `br-ex` bridge as the interface in a machine config manifest file. To update the `br-ex` bridge as a postinstallation task, you must set the `br-ex` bridge as the interface in the NMState configuration of the `NodeNetworkConfigurationPolicy` custom resource (CR) for your cluster. For more information, see xref:../../installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc#creating-manifest-file-customized-br-ex-bridge_ipi-install-post-installation-configuration[Creating a manifest object that includes a customized br-ex bridge] in _Setting up the environment for an OpenShift installation_.
```
Comment (JoeAldinger) [inline networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:23]:
```suggestion
On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the `br-ex` bridge as the interface in a machine config manifest file. To update the `br-ex` bridge as a postinstallation task, you must set the `br-ex` bridge as the interface in the NMState configuration of the `NodeNetworkConfigurationPolicy` custom resource (CR) for your cluster. For more information, see xref:../../installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc#creating-manifest-file-customized-br-ex-bridge_ipi-install-post-installation-configuration[Creating a manifest object that includes a customized br-ex bridge] in "Setting up the environment for an OpenShift installation".
```

Comment (JoeAldinger) [reply networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:23]: I _think_ in ISG we are suppose to use " " for sentence style capitalization when citing internal sources/titles. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks#double-quotation-marks:~:text=Citations%20that%20have%20sentence%2Dstyle%20capitalization%20and%20do%20not%20have%20any%20other%20highlighting

**Review by JoeAldinger — COMMENTED** [review]
Sorry, one small thing.

File: networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc (hunk 1 above)
Comment (dfitzmau) [reply networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:23]: Thanks, Joe. I followed our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute). 

Screenshot from 2024-08-22 11-19-07

Comment (dfitzmau) [reply networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:23]: I think the RH SSG trumps the IBM according to this [excellent pyramid graphic](https://redhat-documentation.github.io/supplementary-style-guide/#_using_style_guides_for_redhat_product_documentation) on the RH SSG.

**Comment by qiowang721** [comment]
Hi @dfitzmau seems the added link is different with the one in the description of [OCPBUGS-38800](https://issues.redhat.com/browse/OCPBUGS-38800)
In the description the link is https://docs.openshift.com/container-platform/4.16/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#creating-manifest-file-customized-br-ex-bridge_ipi-install-installation-workflow , which seems the correct link, could you double confirm?

**Review by Srivaralakshmi — COMMENTED** [review]
LGTM

**Pull request merged**